pkg/printer/printer_test.go

```diff
@@ -60,24 +51,9 @@ func TestPrintWarnings(t *testing.T) {
     — metadata.name: yishay (kind: Pod)
 💡  Suggestion
 
-Failed with yaml validation
```

Comment:
why did we delete this expected output?

Reply:
your correct,
just needed to change the titles to filenames